docs/_includes/content.md

```diff
@@ -91,6 +91,8 @@ Here are some specific examples of how you can use summon with your current tool
 * [Keepass kdbx database file](https://github.com/mskarbek/summon-keepass)
 * [Gopass](https://github.com/gopasspw/gopass-summon-provider)
 * [AWS Secrets Manager](https://github.com/cyberark/summon-aws-secrets)
+* [KeePass 2](https://github.com/stanislavbebej-ext43345/summon-keepass)
+* [Bitwarden Secrets Manager](https://github.com/stanislavbebej-ext43345/summon-secrets-manager)
```

Comment:
I'm curious why you decided to name this repo "summon-secrets-manager" instead of "summon-bitwarden". Given that Conjur is now known as CyberArk Secrets Manager I'm a bit concerned about this being confusing.